bin/civi-download-tools

```diff
@@ -486,7 +486,7 @@ function do_system_focal() {
 
 function do_system_stretch() {
   set -e
-    PACKAGES="acl git wget unzip zip mysql-server mysql-client php7.2-cli php7.2-imap php7.2-ldap php7.2-curl php7.2-mysql php7.2-intl php7.2-gd php7.2-dev php7.2-bcmath php7.2-mbstring php7.2-soap php7.2-zip php7.2-xml apache2 libapache2-mod-php7.2 nodejs"
+    PACKAGES="acl git wget unzip zip mysql-server mysql-client php7.3-cli php7.3-imap php7.3-ldap php7.3-curl php7.3-mysql php7.3-intl php7.3-gd php7.3-dev php7.3-bcmath php7.3-mbstring php7.3-soap php7.3-zip php7.3-xml apache2 libapache2-mod-php7.3 nodejs"
```

Comment:
I think Stretch can be removed? It's pretty old  & unsupported, and does not have PHP 7.2 (unless using 3rd party repos).

Reply:
Stretch is supported - it's the current LTS. We use Ondrej Sury's PHP repos.
![image](https://user-images.githubusercontent.com/2288257/128415356-3d030faf-4815-42d4-859f-6fb50f4c1f5d.png)

Reply:
https://github.com/homotechsual/civicrm-buildkit/blob/aab43bc6e4449402d069b234713ea5fc5a1fe104/bin/civi-download-tools#L500 (For the Sury repo install)

Reply:
oh ok, sounds good :)